lib/features/push_notification/presentation/controller/web_socket_controller.dart

```diff
@@ -32,13 +34,14 @@ class WebSocketController extends PushBaseController {
   WebSocketChannel? _webSocketChannel;
   Timer? _webSocketPingTimer;
   StreamSubscription? _webSocketSubscription;
+  AppLifecycleListener? _appLifecycleListener;
+
+  bool get _isConnectionOpen => _webSocketChannel != null;
```

Comment:
how about if it is non-null, but stream was close?

Reply:
remove it

Reply:
Done